docs/_partials/0-20.migration.mdx

```diff
@@ -0,0 +1,539 @@
+
+import Columns from '@site/src/components/Columns';
+
+
+import Column from '@site/src/components/Column';
+
+import DeployChanges from '../../vcluster/_partials/manage/deploy-changes.mdx'
+
+
+As part of the v0.20.0 release, Loft Labs introduced a new `vcluster.yaml` configuration file with a new structure. This file replaces your existing `values.yaml` configuration file. Previously, configuring
+a virtual cluster had configuration options using CLI flags, anontations and different `values.yaml` options. With the new `vcluster.yaml`, all configuration options are now easily configured in a single source.
+
+<p></p>
+
+:::warning
+Pre-v0.20.0 `values.yaml` files are not compatible with v0.20.0. You need to convert to the new format to use the new vCluster CLI and upgrade existing vCluster instances.
+:::
+
+
+Upgrade vCluster v0.19 to v0.20+ by following these steps:
+
+1. [Upgrade the vCluster CLI](#upgrade-the-cli).
+1. [Convert your Helm values configuration file to the new `vcluster.yaml` configuration file](#convert-your-valuesyaml-to-vclusteryaml).
+1. [Use your new `vcluster.yaml` file to upgrade your vCluster instance](#upgrade-your-vcluster-instance).
+
+## Supported legacy versions
+
+Loft Labs has tested the conversion path from v0.19.x to v0.20.0 and therefore recommends moving to v0.19.x before converting to v0.20.0.
+
+Conversions from vCluster versions prior to 0.19.x might work but have not been tested extensively.
+
+## Before you begin
+
+* Helm version is >= v.3.10.0.
+* Upgrade [vcluster-hostpath-mapper](https://github.com/loft-sh/vcluster-hostpath-mapper) to v0.1.1-beta.1 or later for use with the new vcluster.yaml configuration file in v0.20.
```

Comment:
This makes it sound like you need to install the host path mapper to upgrade, when the intent is to upgrade it only if you're using it.